mne/forward/_field_interpolation.py

```diff
@@ -193,6 +138,170 @@ def make_surface_mapping(info, surf, ch_type='meg', trans=None, mode='fast',
+
+    trans_fnames = glob.glob(os.path.join(subjects_dir, subject,
+                                          '*-trans.fif'))
+    if len(trans_fnames) < 1:
+        raise RuntimeError('Could not find the transformation for '
+                           '{subject}'.format(subject=subject))
+    elif len(trans_fnames) > 1:
+        raise RuntimeError('Found multiple transformations for '
+                           '{subject}'.format(subject=subject))
+    return trans_fnames[0]
+
+
+def make_field_map(evoked, trans_fname='auto', subject=None, subjects_dir=None,
+                   ch_type=None, mode='fast', n_jobs=1):
+    """Compute surface maps used for field display in 3D
+
+    Parameters
+    ----------
+    evoked : Evoked | Epochs | Raw
+        The measurement file. Need to have info attribute.
+    trans_fname : str | 'auto' | None
+        The full path to the `*-trans.fif` file produced during
+        coregistration. If present or found using 'auto'
+        the maps will be in MRI coordinates.
+        If None, map for EEG data will not be available.
+    subject : str | None
+        The subject name corresponding to FreeSurfer environment
+        variable SUBJECT. If None, map for EEG data will not be available.
+    subjects_dir : str
+        The path to the freesurfer subjects reconstructions.
+        It corresponds to Freesurfer environment variable SUBJECTS_DIR.
+    ch_type : None | 'eeg' | 'meg'
+        If None, a map for each available channel type will be returned.
+        Else only the specified type will be used.
+    mode : str
+        Either `'accurate'` or `'fast'`, determines the quality of the
+        Legendre polynomial expansion used. `'fast'` should be sufficient
+        for most applications.
+    n_jobs : int
+        The number of jobs to run in parallel.
+
+    Returns
+    -------
+    surf_maps : list
+        The surface maps to be used for field plots. The list contains
+        separate ones for MEG and EEG (if both MEG and EEG are present).
+    """
+    info = evoked.info
+
+    if ch_type is None:
+        types = [t for t in ['eeg', 'meg'] if t in evoked]
+    else:
+        if ch_type not in ['eeg', 'meg']:
+            raise ValueError("ch_type should be 'eeg' or 'meg' (got %s)"
+                             % ch_type)
+        types = [ch_type]
+
+    if trans_fname == 'auto':
+        # let's try to do this in MRI coordinates so they're easy to plot
+        trans_fname = _find_trans(subject, subjects_dir)
+
+    if 'eeg' in types and trans_fname is None:
+        print('No trans file available. EEG data ignored.')
+        types.remove('eeg')
+
+    if len(types) == 0:
+        raise RuntimeError('No data available for mapping.')
+
+    trans = None
+    if trans_fname is not None:
+        trans = read_trans(trans_fname)
+
+    surfs = []
+    for this_type in types:
+        if this_type == 'meg':
+            surf = get_meg_helmet_surf(info, trans)
```

Comment:
might be a bit late to the party, but why do we need to pass `trans` here if we need everything in head coordinates?